src/types/types.d.ts

```diff
@@ -168,3 +168,8 @@ export interface PaymentFormData {
 export interface AnyObject {
   [key: string]: any;
 }
+
+export enum StellarThemeValue {
+  LIGHT = "light-mode",
+  DARK = "dark-mode",
+}
```

Comment:
In the future, this type could be exported from https://github.com/stellar/stellar-design-system, WDYT?

Reply:
That's a good idea, @marcelosalloum. I'll add this to the list. 👍